src/lib/resource/sections/rep-confirmation-method.js

```diff
@@ -0,0 +1,247 @@
+'use strict';
+
+module.exports = {
+    section: {
+        schema: {
+            $schema: 'http://json-schema.org/draft-07/schema#',
+            type: 'object',
+            propertyNames: {
+                enum: [
+                    'q-rep-confirmation-method',
+                    'q-rep-enter-your-email-address',
+                    'q-rep-enter-your-telephone-number'
+                ]
+            },
+            properties: {
+                'q-rep-confirmation-method': {
+                    title: "How should we tell you we've got the application?",
+                    type: 'string',
+                    oneOf: [
+                        {
+                            title: 'Email',
+                            const: 'email'
+                        },
+                        {
+                            title: 'Text message',
+                            const: 'text'
+                        },
+                        {
+                            title: "I don't have an email address or UK mobile phone number",
+                            const: 'none'
+                        }
+                    ],
+                    meta: {
+                        classifications: {
+                            theme: 'rep-details'
+                        },
+                        summary: {
+                            title: 'Confirmation method'
+                        }
+                    }
+                },
+                'q-rep-enter-your-email-address': {
+                    type: 'string',
+                    title: 'Email address',
+                    maxLength: 50,
+                    format: 'email',
+                    errorMessage: {
+                        maxLength: 'Email address must be 50 characters or less',
+                        format:
+                            'Enter an email address in the correct format, like name@example.com'
+                    },
+                    meta: {
+                        classifications: {
+                            theme: 'main-applicant-details'
```

Comment:
@armoj should this theme be 'rep-details'

Reply:
fixed